src/wasm-lib/kcl/src/execution/state.rs

```diff
@@ -167,11 +174,12 @@ impl ExecState {
         self.global.path_to_source_id.get(path).cloned()
     }
 
-    pub(super) fn add_module(&mut self, id: ModuleId, path: ModulePath, repr: ModuleRepr) {
+    pub(super) fn add_path_to_source_id(&mut self, path: ModulePath, id: ModuleId) {
         debug_assert!(!self.global.path_to_source_id.contains_key(&path));
```

Comment:
Now that these functions are split in two, consider adding a new debug_assert in `add_module()`.